packages/utils/typescript/lib/generators/content-types/index.js

```diff
@@ -23,6 +28,12 @@ const generateContentTypesDefinitions = async (options = {}) => {
     definition: models.schema.generateSchemaDefinition(contentType),
   }));
 
+  options.logger.debug(`Found ${contentTypesDefinitions.length} content-types.`);
```

Comment:
```suggestion
  options.logger.debug(`Found ${contentTypesDefinitions.length} content-type${contentTypesDefinitions.length === 1 ? '' : 's'}.`);
```
not sure if we care or not

Reply:
I think it adds a lot of verbosity for very little benefit (those are debug logs anyway)